docs/api/security/security.md

```diff
@@ -2,10 +2,14 @@
 
 Security on Arm Mbed OS is divided into the following parts:
```

Comment:
@alzix, on this page, we said we wanted to "Add intro blurb about PSA standard and a link to the PSA page." What exactly do we want to say about the PSA standard?

Reply:
I updated this page with the content from the PSA page.